apps/nextjs-website/src/_contents/appIo/tutorialLists.ts

```diff
@@ -5,6 +5,16 @@ import { Tutorial } from '@/lib/types/tutorialData';
 import { appIoBannerLinks } from '@/_contents/appIo/bannerLinks';
 
 export const appIoTutorials: readonly Tutorial[] = [
+  {
+    title: 'Come utilizzare correttamente servizi e messaggi in app IO',
+    path: `${appIoTutorialListsPath.path}/come-utilizzare-correttamente-servizi-e-messaggi-in-app-io`,
+    name: 'Come utilizzare correttamente servizi e messaggi in app IO',
+    image: {
+      url: '/images/app-io-messaggi.png',
+      alternativeText: 'Immagine: Servizi e messaggi in app IO',
+    },
+    showInOverview: true,
+  },
```

Comment:
Why "SERVIZIO" and "MESSAGGIO" are not part of the header content of the table?
![image](https://github.com/pagopa/developer-portal/assets/39835990/1103e763-ad8f-4701-9d6a-9209781267ff)

Reply:
Why there's a back slash at the end of the page?
![image](https://github.com/pagopa/developer-portal/assets/39835990/db09f93e-aed2-457f-8af1-f118dd5a137c)

Reply:
The header content of the table is not rendered in our frontend, resulting in the loss of the header row, that's why it's been skipped.

Reply:
Both solved in gitbook
<img width="1439" alt="image" src="https://github.com/pagopa/developer-portal/assets/122867940/0aa1c29e-1b0b-4093-910b-c6247f70a787">